etc/flux.service.in

```diff
@@ -21,6 +21,7 @@ ExecStart=/bin/bash -c '\
   -Sbroker.quorum=0 \
   -Sbroker.quorum-timeout=none \
 '
+SyslogIdentifier=flux-broker
```

Comment:
Cool!

Since the name of the service is `flux` and the journal will contain output from more than just the broker, should the identifier be `flux` instead of `flux-broker`?

I'm not sure, just asking.

Reply:
I actually was debating that before making the change.  I eventually went with `flux-broker` solely out of a "well lets keep the program name just for consistency".  But `flux` would be fine as well. @grondo, would you like to put in your vote?

Reply:
Now that you mention it, I guess it would be the process not the service here normally.  Meh, I withdraw my suggestion :-)

Reply:
I slightly prefer `flux` since it is shorter and the unit name is `flux`, but I could go either way as well. Either one is better than `bash` :slightly_smiling_face: